infra/auth-logic/src/main/java/com/simprints/infra/authlogic/authenticator/SignerManager.kt

```diff
@@ -35,14 +35,14 @@ internal class SignerManager @Inject constructor(
         try {
             // Store Firebase token so it can be used by ConfigManager
             authStore.storeFirebaseToken(token)
-            configRepository.refreshProject(projectId)
+            configManager.refreshProject(projectId)
```

Comment:
Wouldn't it be possible to simply add the `tokenizeExistingRecords(project)` here similar to the `DebugFragment` instead of adding back the whole module?

Reply:
Unfortunately, it will not be enough due to `refreshProject` being called from the `ConfigManager::getProjectConfiguration` (previously `ConfigRepository::getProjectConfiguration`). The `::getProjectConfiguration` is called in 100+ places in the project, and we need to ensure that the untokenized records are encrypted any time the project is successfully refreshed.
![image](https://github.com/Simprints/Android-Simprints-ID/assets/129499142/ca41f4de-331a-4074-93a1-020360a09cce)